jenkins-pipelines/oss/tier1/longevity-multidc-schema-topology-changes-ipv6-12h.jenkinsfile

```diff
@@ -0,0 +1,14 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+longevityPipeline(
+    backend: 'aws',
+    availability_zone: 'c',
+    region: '''["eu-west-1", "eu-west-2"]''',
+    test_name: 'longevity_test.LongevityTest.test_custom_time',
+    test_config: '''["test-cases/longevity/longevity-multidc-parallel-topology-schema-changes-12h.yaml", "configurations/network_config/all_addresses_ipv6_public.yaml"]''',
```

Comment:
can you move this one to the current case ? i.e. so we don't need to start messing around with triggering ?

Reply:
Do you mean - do not create new pipeline? Convert existing to be IPv6?

Reply:
yes

Reply:
@fruch changed